### DISCUSSION TITLE
posts/adding-giscus-comments-to-my-blog/

### DISCUSSION THREAD

**Discussion by giscus** [MartinP7r/blog#3] [Posts]
# posts/adding-giscus-comments-to-my-blog/

Giscus is a service that enables you to have discussion-like comments and reactions on your website using GitHub’s Discussion feature as a backend.

https://blog.martinp7r.com/posts/adding-giscus-comments-to-my-blog/

**Comment by ic4rta** [comment]
thaanks

**Comment by triccsr** [comment]
thanks

**Comment by seletz** [comment]
Thanks, added to my own [blog](https://seletz.github.io/posts/giscus-comments)!

**Comment by danywebxx** [comment]
Thanks! Your post help me to do the same!

**Comment by bericontraster** [comment]
Thanks bro that really helped me.

**Comment by nymanska** [comment]
Thanks Martin. This inspired me to add Giscus on my own site and I referenced you on my post as well 👍
